tests/unit/models/BucketInfo.spec.js

```diff
@@ -292,8 +293,16 @@ Object.keys(acl).forEach(
                         dummyBucket._objectLockConfiguration,
                     notificationConfiguration: dummyBucket._notificationConfiguration,
                     tags: dummyBucket._tags,
-                    capabilities: dummyBucket._capabilities,
-                    quotaMax: dummyBucket._quotaMax,
+                    capabilities: dummyBucket._capabilities ? {
+                        ...dummyBucket._capabilities,
+                        VeeamSOSApi: dummyBucket._capabilities.VeeamSOSApi ? {
+                            ...dummyBucket._capabilities.VeeamSOSApi,
+                            CapacityInfo: BucketInfo.serializeCapacityInfo(
+                                dummyBucket._capabilities.VeeamSOSApi.CapacityInfo,
+                            ),
+                        } : undefined,
```

Comment:
we have this multiple times, probably should have another dedicated function `BucketInfo.serializeCapabilities()` or `Capabilities.serialize()`